languages.toml

```diff
@@ -2091,6 +2091,20 @@ language-servers = [ "jsonnet-language-server" ]
 name = "jsonnet"
 source = { git = "https://github.com/sourcegraph/tree-sitter-jsonnet", rev = "0475a5017ad7dc84845d1d33187f2321abcb261d" }
 
+[[language]]
+name = "ada"
+scope = "source.ada"
+injection-regex = "ada"
+file-types = ["adb", "ads"]
+roots = ["alire.toml"]
```

Comment:
Is it possible to add wildcards here? Ada supports `*.gpr` files in a form of `PROJECT_NAME.gpr` (`gpr` = GNAT Project).

Reply:
Not currently, but this would be useful for other languages (dotnet `PROJECT_NAME.sln` comes to mind)